sucuri.php

```diff
@@ -8,7 +8,7 @@
  * Author: Sucuri Inc.
  * Text Domain: sucuri-scanner
  * Domain Path: /lang
- * Version: 1.8.36
+ * Version: 1.8.37
```

Comment:
Should we also update the `sucuri.php` and the `readme.txt` with the new version?